.copier-answers.yml

```diff
@@ -0,0 +1,8 @@
+# Changes here will be overwritten by Copier
+_commit: v0.2
+_src_path: gh:LSSTDESC/RAIL-project-template
+author_email: later@later.com
```

Comment:
`later@gator.com`? `afterawhile@crocodile.net`?

Reply:
Where does that actually end up?

Reply:
I left a comment in the pyproject.toml file pointing out where it's used. And if you want to up update it, it should be safe to do so without running `copier update` again as long as you change it in both .copier-answers.yml and pyproject.toml.